### ISSUE TITLE
Cron Job

### ISSUE THREAD

**Issue by pjebs** [bshaffer/oauth2-server-php#533]
Is there any built in function that we can execute in a CRON job to clean up the databases so we can get rid of old tokens etc and any data that is no longer relevant?

**Comment by bshaffer** [comment]
no, this functionality does not exist. It would be a good addition to the storage classes.

**Comment by pjebs** [comment]
I have sent an email to bs***@gmail.com with some code.

**Comment by pjebs** [comment]
I have sent an email to bs***@gmail.com with some code addressing this!
